doc/build.md

```diff
@@ -12,37 +12,32 @@ What's working so far?
 
 ## Requirements
 
-See [Requirements on the wiki](https://github.com/mimblewimble/docs/wiki/Building).
-
 But basically:
 - rust 1.21+ (use [rustup]((https://www.rustup.rs/))- i.e. `curl https://sh.rustup.rs -sSf | sh; source $HOME/.cargo/env`)
 - cmake 3.2+ (for [Cuckoo mining plugins]((https://github.com/mimblewimble/cuckoo-miner)))
 - rocksdb + libs for compiling rocksdb:
   - clang (clanglib or clang-devel or libclang-dev)
   - llvm (Fedora llvm-devel, Debian llvm-dev)
-- ncurses and libs (ncurses ncurses5w)
+- ncurses and libs (ncurses, ncurses5w)
 - linux-headers (reported needed on Alpine linux)
 
 
 ## Build steps
 
 ```sh
-    git clone https://github.com/mimblewimble/grin.git
-    cd grin
-    # Decide yourself if you want master, another branch, or tag
-    cargo update # first time, in case you have some old stuff
-    cargo build
-    # or: cargo install && cargo clean # if you don't plan to build again soon
+git clone https://github.com/mimblewimble/grin.git
+cd grin
+cargo update
```

Comment:
I suggest we remove the `cargo update` step, especially after having checked in a curated and tested working Cargo.lock.

In the Lobby today we had this guy who did cargo update and got updated to slog 2.2 and then couldn't build. Got a copy of my Cargo.lock and proceeded to do cargo update as that's what the instructions said :). Sometimes less is more.